src/shared/locales/pt-BR/translation.json

```diff
@@ -307,7 +309,7 @@
         "paperWallet": "Carteira em Papel",
         "writeYourSeedDown": "Anotar Seed",
         "seedClearedExplanation": "Para sua segurança, a Seed foi removida da área de transferência.",
-        "letsWriteDownYourSeed": "Vamos gravar sua seed",
+        "letsWriteDownYourSeed": "Vamos salvar sua seed",
```

Comment:
Please don't make manual changes to the translation files (except `locales/en/translation.json`). These are automatically generated and will likely be overwritten when translations are synced from [Crowdin](https://crowdin.com/project/iota-trinity-wallet). Instead, you can suggest changes there and they will automatically be synced back to this repository.